source/components/triggering-changes-with-actions.md

```diff
@@ -156,3 +156,56 @@ a function instead.
 Actions in components allow you to
 decouple an event happening from how it's handled, leading to modular,
 more reusable components.
+
+## Calling Actions Across Component Layers
+
+When your components go multiple layers deep, its common to need to handle an action several layers up the tree.
```

Comment:
`When your components go multiple template layers deep, its common to need to handle an action several layers up the tree.`
Difficult to express the difference.  "template layers" hopefully will give the impression of multiple templates in use.  Hopefully the reader also gets it from the example...